ArduPlane/quadplane.cpp

```diff
@@ -4064,6 +4094,14 @@ bool QuadPlane::in_vtol_takeoff(void) const
 // called when we change mode (for any mode, not just Q modes)
 void QuadPlane::mode_enter(void)
 {
+    // set rate shaping time constants
+    attitude_control->set_roll_pitch_rate_tc(acro_rp_rate_tc);
+    if (plane.control_mode == &plane.mode_qacro) {
+        attitude_control->set_yaw_rate_tc(acro_y_rate_tc);
+    } else {
+        attitude_control->set_yaw_rate_tc(yaw_rate_tc);
+    }
```

Comment:
try moving this down inside the `available()` check as for pos_control Right at boot attitude_control will still be a nullptr.